cockroachcloud/managing-cmek.md

```diff
@@ -96,7 +159,90 @@ curl --request GET \
   --header "Authorization: Bearer ${API_KEY}"
 ```
 
-## Revoking CMEK for a cluster
+## Rotate a CMEK key
+
+A CMEK key can be rotated within your KMS platform or within {{ site.data.products.dedicated }}.
+
+- When you rotate a CMEK key within your KMS platform, a new version of the key is created with new key material. In both GCP KMS and AWS KMS, encryption operations automatically uses the active key version, while decryption operations automatically use the key version that was used to encrypt the data. For this reason, {{ site.data.products.dedicated }} does not need any awareness of rotation operations within your KMS platform.
+
+- When you rotate a CMEK key using the {{ site.data.products.db }} API, you update your cluster's configuration to begin using a different CMEK key for one or more regions. {{ site.data.products.dedicated }} begins using the new key to protect the store key on nodes in that cluster region. In a similar way, after adding a new region to a cluster, you can begin using CMEK to protect data at rest in that region by "rotating" that region from using no CMEK key to using a CMEK key.
+
+The API to rotate a CMEK key is nearly identical to the API for [activating CMEK on a cluster](#step-4-activate-cmek-for-your-cockroachdb-dedicated-cluster), with one notable exception. When you activate CMEK, you use a `POST` request that includes a CMEK key for each of the cluster's regions. When you rotate a CMEK key, you use a `PUT` request and include a CMEK key for each region whose CMEK key to rotate.
```

Comment:
Comms around this are somewhat tricky until the next prod release, and i'm not sure how we want to navigate them:

* Right now: `PUT` requests _are not_ allowed on non-`ENABLED` clusters. `PUT` is used for exclusively rotation, not initial-enablement. (I guess PUT-for-add-region is a divergence addressed in other comments).
* After next week: `PUT` requests _are_ allowed for non-`ENABLED` clusters. However, `PUT` for initial-enablement must have all regions.

Reply:
Discussed offline and we will update the docs when the new behavior is available. cc/ @MichaelTrestman